packages/customize-widgets/src/components/welcome-guide/index.js

```diff
@@ -22,7 +21,20 @@ export default function WelcomeGuide( { sidebar } ) {
 
 	return (
 		<div className="customize-widgets-welcome-guide">
-			<EditorImage />
+			<img
```

Comment:
Why inlining all these images when we can just change the `src` attributes?

Reply:
I figured there's no real benefit to having a seperate `images.js` file now that there's no massive lines of base64 data.